### PR TITLE
Time service test fix (+-5 miliseconds reserve added)

### LINKED ISSUE #172: Fix Time service test
When testing currentDate(), this is not a good approach:
```
assertThat(timeService.currentDate(), is(equalTo(new Date())));
```
because of reasons, I told you that time. try to figure out how to test this better, eg. set some acceptable difference, let's say 5ms and check whether times differ by that amount. 
Don't forget to put a comment there, why it is made like so.

Fix ASAP please, because now the behavior of tests is very "undefined"

### PR DESCRIPTION
**Pull request by Ahimtar**
Resolves #172

### CODE REVIEW

File: service/src/test/java/com/kodemon/service/test/TimeServiceTest.java [hunk 1]
```diff
@@ -28,7 +29,8 @@
 
     @Test
     public void currentDateTest() {
-        assertThat(timeService.currentDate(), is(equalTo(new Date())));
+        assertThat(new Date().getTime()-5L, is(lessThan(timeService.currentDate().getTime())));
```
Comment (mseleng) [inline service/src/test/java/com/kodemon/service/test/TimeServiceTest.java]: nonononono :smile: 
Look, you are basically creating 4 different times here:
1. when you call `new Date().getTime()-5L`
2. when you call `timeService.currentDate().getTime()`
3. when you call `timeService.currentDate().getTime()` 
4. when you call `new Date().getTime()+5L`

Comment (mseleng) [inline service/src/test/java/com/kodemon/service/test/TimeServiceTest.java]: You should it somehow like this: 
```
long myDate = new Date().getTime();
long serviceDate = timeService.currentDate().getTime();
assertThat(myDate...);
...
```

Comment (Ahimtar) [inline service/src/test/java/com/kodemon/service/test/TimeServiceTest.java]: :( (y)

File: service/src/test/java/com/kodemon/service/test/TimeServiceTest.java [hunk 2]
```diff
@@ -28,7 +29,10 @@
 
     @Test
     public void currentDateTest() {
-        assertThat(timeService.currentDate(), is(equalTo(new Date())));
+        long myDate = new Date().getTime();
+        long serviceDate = timeService.currentDate().getTime();
+        assertThat(myDate-5L, is(lessThan(serviceDate)));
```
Comment (mseleng) [inline service/src/test/java/com/kodemon/service/test/TimeServiceTest.java]: my bad.
Please do `Math.abs(myDate - serviceDate) < 5L`, but extract 5L to the static final constant pls.

File: service/src/test/java/com/kodemon/service/test/TimeServiceTest.java [hunk 3]
```diff
@@ -29,10 +29,10 @@
 
     @Test
     public void currentDateTest() {
+        long allowedReserve = 5L;
```
Comment (mseleng) [inline service/src/test/java/com/kodemon/service/test/TimeServiceTest.java]: I said static final constant and this is non of that :smile:. But merge it :smile:

**Pull request merged**